egs/ami/s5/run_prepare_shared.sh

```diff
@@ -3,41 +3,15 @@
 . ./cmd.sh
 . ./path.sh
 
-# To run this script you need SRILM,
 
 # Path to Fisher transcripts LM interpolation (if not defined only AMI transcript LM is built),
-FISHER_TRANS=`pwd`/eddie_data/lm/data/fisher/part1 # Edinburgh, [DEFAULT]
-# Path where AMI gets downloaded (or where locally available),
-AMI_DIR=$PWD/DOWNLOAD/amicorpus # [DEFAULT]
+#FISHER_TRANS=`pwd`/eddie_data/lm/data/fisher/part1 # Edinburgh,
+#FISHER_TRANS=/export/corpora4/ami/fisher_trans/part1 # JHU,
+#FISHER_TRANS=/mnt/matylda2/data/FISHER/fe_03_p1_tran # BUT,
+FISHER_TRANS=/export/ws15-ffs-data/corpora/LDC/LDC2004T19/fe_03_p1_tran # JSALT2015 workshop, cluster AWS-EC2,
 
```

Comment:
Yes I know, but this is not applicable to the case when no domain is set like in the 'AWS' cluster. AFAIK, manually commenting/uncommenting or eventually adding a path is a better solution. On the other hand. Both approaches can be combined...